system/expressionengine/third_party/twitter/mod.twitter.php

```diff
@@ -143,6 +143,39 @@ public function search()
 		return $this->render_tweets($statuses, $prefix, $userprefix);
 	}
 
+	public function getbyid()
```

Comment:
Would this be better as `{exp:twitter:tweet id="..."}` or `{exp:twitter:single_tweet id="..."}`? `{exp:twitter:getbyid}` doesn't seem to have much flow to it. What am I getting by ID?